### PR TITLE
Fix typo in mk-fipsmodule-cnf.pl

### PR DESCRIPTION
**Pull request by 0140454**
I think I made them broken when editing multiple lines at the same time.

##### Checklist

- [x] documentation is added or updated
- [x] tests are added or updated

### CODE REVIEW

**Review by paulidale — APPROVED** [review]

**Review by slontis — APPROVED** [review]
Not overly concerned about this.
I need to do a pass after 'everything' is merged, to make sure this file is correct.

File: util/mk-fipsmodule-cnf.pl [hunk 1]
```diff
@@ -50,9 +50,9 @@
 drbg-no-trunc-md = $drgb_no_trunc_dgst
 module-mac = $module_mac
 hkdf-digest-check = $kdf_digest_check
-tls13-kdf-digest-check = $kd_digest_check
+tls13-kdf-digest-check = $kdf_digest_check
```
Comment (tom-cosgrove-arm) [inline util/mk-fipsmodule-cnf.pl:53]: @paulidale @slontis Any reason we don't have `use strict` in this file?

**Review by tom-cosgrove-arm — APPROVED** [review]

File: util/mk-fipsmodule-cnf.pl (hunk 1 above)
Comment (paulidale) [reply util/mk-fipsmodule-cnf.pl:53]: No idea.  @levitte?

**Review by shahsb — APPROVED** [review]

**Comment by slontis** [comment]
Agree its urgent to fix other PR's.

**Comment by paulidale** [comment]
Merged, thanks for the fix.

**Comment by junaruga** [comment]
Why couldn't this repository's CI find this issue?

**Comment by tom-cosgrove-arm** [comment]
> Why couldn't this repository's CI find this issue?

It would have been found very quickly with `use strict;`, hence my question. But yes, there's also the issue that testing in the CI didn't spot this, but then no CI covers 100%

**Comment by junaruga** [comment]
> > Why couldn't this repository's CI find this issue?
> 
> It would have been found very quickly with `use strict;`, hence my question. But yes, there's also the issue that testing in the CI didn't spot this, but then no CI covers 100%

I see. With this issue, the broken `fipsmodule.cnf`, OpenSSL even couldn't load the fips provider, and I reported the issue at <https://github.com/openssl/openssl/discussions/24991>. So, I thought some test cases were missing in the CI.

**Pull request closed without merging**